crates/release_plz/src/commands/completions.rs

```diff
@@ -1,20 +1,25 @@
+use std::io::stdout;
+
 use clap::CommandFactory;
 use clap_complete::Shell;
 
+use crate::CliArgs;
+
+/// Generate command autocompletions for various shells.
 #[derive(clap::Parser, Debug)]
-pub struct GenerateCompletions {
+pub struct Completions {
     /// Shell option
     #[arg(default_value = "bash")]
     shell: Shell,
 }
 
-impl GenerateCompletions {
-    pub fn print(&self) {
+impl Completions {
+    pub fn run(&self) {
```

Comment:
Renamed for consistency with the other commands, not a big deal. It could be left as-is.